app/services/proofing/lexis_nexis/ddp/proofer.rb

```diff
@@ -55,12 +57,20 @@ def build_result_from_response(verification_response)
           request_result = body['request_result']
           review_status = body['review_status']
 
+          validate_review_status!(review_status)
+
           result.review_status = review_status
           result.add_error(:request_result, request_result) unless request_result == 'success'
           result.add_error(:review_status, review_status) unless review_status == 'pass'
 
           result
         end
+
+        def validate_review_status!(review_status)
+          return if VALID_REVIEW_STATUSES.include?(review_status)
+
+          raise "Unexpected ThreatMetrix review_status value: #{review_status}"
```

Comment:
[question] is this "unexpected" in the sense that truly nothing should trip this or "unexpected" in the sense that something expected could trip this but only inappropriately? (thinking explicitly of the `no_result` that we previously thought could happen)

Reply:
We can consider changing the text here. Do you have a suggestion for an improved message?